docs/tasks/logs.rst

```diff
@@ -0,0 +1,259 @@
+
+Confirm that the Loki StatefulSet is successfully rolled out:
+
+.. code-block:: bash
+
+  kubectl -n loki rollout status sts/loki
+
+.. code-block:: bash
+
+  statefulset rolling update complete 1 pods at revision loki-6d7b88b95c...
+
+.. note::
+  The Loki configuration used in this installation is meant for demonstration
+  purposes only. The Helm chart deploys a non-HA single instance of Loki,
+  managed by a StatefulSet workload. See the `Loki installation documentation`_
+  for other installation methods that may be more suitable for your
+  requirements.
+
+Use Helm to install Grafana with a pre-configured Loki data source:
+
+.. code-block:: bash
+
+  svc_url=$(kubectl -n loki get svc loki -ojsonpath='{.metadata.name}.{.metadata.namespace}:{.spec.ports[?(@.name=="http-metrics")].port}')
+
+  cat <<EOF | helm -n grafana install --create-namespace grafana grafana/grafana -f -
+  datasources:
+    datasources.yaml:
+      apiVersion: 1
+      datasources:
+      - name: Loki
+        type: loki
+        url: http://$svc_url
+        access: proxy
+        isDefault: true
+  EOF
+
+Confirm that the Grafana Deployment is successfully rolled out:
+
+.. code-block:: bash
+
+  kubectl -n grafana rollout status deploy/grafana
+
+.. code-block:: bash
+
+  deployment "grafana" successfully rolled out
+
+Set up port-forward to access the Grafana UI:
+
+.. code-block:: bash
+
+  kubectl -n grafana port-forward svc/grafana 3000:80
+
+Use a web browser to navigate to ``localhost:3000``:
+
+.. image:: img/logs-grafana-login.png
+
+The default login username is ``admin``.
+
+The login password can be retrieved using the following command:
+
+.. code-block:: bash
+
+  kubectl -n grafana get secret grafana -o jsonpath="{.data.admin-password}" | base64 --decode ; echo
+
+Navigate to the data sources configuration under ``Configuration`` >
+``Data Sources`` using the left-hand panel.
+
+Confirm that the ``Loki`` data source have already been added as part of the
+Grafana installation:
+
+.. image:: img/logs-grafana-data-source.png
+
+Access the ``Loki`` data source configuration page.
+
+Use the ``Test`` button near the bottom of the page to test the connectivity
+between Grafana and Loki:
+
+.. image:: img/logs-grafana-loki-test.png
+
+The final step in the setup involves installing Promtail. Promtail is an agent
+that can be used to discover log targets and stream their logs to Loki:
+
+.. code-block:: bash
+
+  svc_url=$(kubectl -n loki get svc loki -ojsonpath='{.metadata.name}.{.metadata.namespace}:{.spec.ports[?(@.name=="http-metrics")].port}')
+
+  helm -n loki upgrade --install --create-namespace promtail grafana/promtail \
+    --set image.tag=2.5.0 \
+    --set "config.clients[0].url=http://${svc_url}/loki/api/v1/push"
+
+Confirm that the Promtail DaemonSet is successfully rolled out:
+
+.. code-block:: bash
+
+  kubectl -n loki rollout status ds/promtail
+
+.. code-block:: bash
+
+  daemon set "promtail" successfully rolled out
```

Comment:
same as above, maybe in the same code block?

Reply:
Got it - I removed them since they aren't necessary.